packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -28,29 +29,29 @@ declare global {
  * @slot actions-start - A slot for adding `calcite-action`s or content to the start side of the component's header.
  */
 export class AccordionItem extends LitElement {
-  // #region Static Members
+  //#region Static Members
```

Comment:
nitpick: Was this result of linting? I wonder if dev has the latest changes here?

Reply:
Hm... Yeah, it's linting. I always update the dev before starting a new branch.